content/docs/introducing-jsx.md

```diff
@@ -175,10 +184,12 @@ const element = {
 };
 ```
 
-These objects are called "React elements". You can think of them as descriptions of what you want to see on the screen. React reads these objects and uses them to construct the DOM and keep it up to date.
+Deze objecten worden "React elementen" genoemd.
+Je kunt ze zien als beschrijvingen van wat je wilt zien op het scherm.
+React leest deze objecten en gebruikt ze om het DOM op te bouwen and bij te werken.
```

Comment:
Is it "de" or "the" DOM? I always thought it was "de DOM" 😆

Reply:
Well, it's an abbreviation for Document Object Model and it's "het model". But I always say "de DOM" as well 😄